app/admin/login/page.tsx

```diff
@@ -0,0 +1,295 @@
+"use client";
```

Comment:
You have changed the routing for the admin login.  Stick to the current implementation.